From 60de2946eb356ceae1fcf24c9155548660c5548a Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Tue, 1 Sep 2015 15:34:38 +0800
Subject: [PATCH 0385/5242] MLK-11480-02 ARM: dts: add sai<->wm8962 sound card
 support for imx6sx-sdb board

commit  8ef0c58f77feae58eb99bd2e5605117ff9fbb67b from
https://source.codeaurora.org/external/imx/linux-imx.git

add sai<->wm8962 sound card support for imx6sx-sdb board

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb-sai.dts |    2 ++
 arch/arm/boot/dts/imx6sx.dtsi        |    1 +
 2 files changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb-sai.dts b/arch/arm/boot/dts/imx6sx-sdb-sai.dts
index 2ac865b..c1a7b325 100644
--- a/arch/arm/boot/dts/imx6sx-sdb-sai.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb-sai.dts
@@ -53,6 +53,8 @@
 };
 
 &sai1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_sai1>;
 	status = "okay";
 };
 
diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index c3d3647..1407e7e 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -1110,6 +1110,7 @@
 				clock-names = "bus", "mclk1", "mclk2", "mclk3";
 				dma-names = "rx", "tx";
 				dmas = <&sdma 31 24 0>, <&sdma 32 24 0>;
+				dma-source = <&gpr 0 15 0 16>;
 				status = "disabled";
 			};
 
-- 
1.7.9.5

